### DISCUSSION TITLE
How to double-click another element in the canvas to open asset management

### DISCUSSION THREAD

**Discussion by zhangdongxu-web** [GrapesJS/grapesjs#4426] [Q&A] [answered]
Excuse me how to **double-click the video element in the canvas to start asset management**, just like the default double-click img element in grapesjs to open the asset manager. Thank you for your help  I look forward to your help

**Comment by artf** [comment, marked as answer]
Hi @zhangdongxu-web you can extend the Video component view by following the same behavior from the Image component.
Here you bind the event
https://github.com/artf/grapesjs/blob/22fb44ba9934f87ce12f60fa2a7b37b6ec37835b/src/dom_components/view/ComponentImageView.js#L10
and then you add the onActive method
https://github.com/artf/grapesjs/blob/22fb44ba9934f87ce12f60fa2a7b37b6ec37835b/src/dom_components/view/ComponentImageView.js#L61-L77

**Reply by zhangdongxu-web** [reply]
Thank you for your answer. **This method is very useful**
